R/options.R

```diff
@@ -206,8 +200,6 @@ dyOptions <- function(dygraph,
     options$colors <- as.list(colors)
   options$colorValue <- colorValue
   options$colorSaturation <- colorSaturation
-  options$drawXAxis <- drawXAxis
-  options$drawYAxis <- drawYAxis
```

Comment:
Here we're breaking existing code that passes those values. The values need to be set to the correct spot within the individual axis options (or you could drop this code in favor of the other PR).